image-clipboard.html

```diff
@@ -0,0 +1,436 @@
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: 60px;
+        background-color: transparent;
+        border: 3px solid #02576c;
+        box-shadow: none;
+        transition: background-color .2s, box-shadow .2s;
+      }
+      .add-btn:active {
+        border: 3px solid #02576c;
+        background-color: rgba(2, 87, 108, 0.5);
+      }
+      .corner-btn {
+        width: 36px;
+        height: 36px;
+        -webkit-appearance: none;
+        border: none;
+        background: none;
+        margin: 5px;
+        position: absolute;
+        pointer-events: none;
+        opacity: 0;
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: 24px;
+        transition: opacity .2s;
+        cursor: pointer;
+        border-radius: 50%;
+        background-color: rgba(0,0,0,0.5);
+        fill: #ffe837;
+      }
+      .card:hover .corner-btn,
+      .card:focus-visible .corner-btn {
+        pointer-events: all;
+        opacity: 0.8;
+      }
+      .card:hover .corner-btn:hover,
+      .corner-btn:focus-visible {
+        opacity: 1;
+      }
+      .remove-btn {
+        /* Material remove icon */
+        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='24px' viewBox='0 0 24 24' width='24px' fill='%23ffe837'%3E%3Cpath d='M0 0h24v24H0V0z' fill='none'/%3E%3Cpath d='M19 13H5v-2h14v2z'/%3E%3C/svg%3E");
+        top: 0;
+        right: 0;
+      }
+      .placeholder {
+        box-shadow: none;
+        pointer-events: none;
+        background: none;
+        border: 3px solid #05A19C;
+      }
+      .dragging {
+        cursor: grabbing;
+        user-select: none;
+        touch-action: none;
+      }
+      .card.dragged {
+        position: fixed;
+        cursor: unset;
+        box-shadow: 0 6px 18px rgba(0, 0, 0, 0.6);
+        z-index: 100;
+      }
+      .card.dragged .corner-btn {
+        opacity: 0;
+      }
+      :focus {
+        outline: none;
+      }
+      :focus-visible {
+        box-shadow: 0 0 0 3px #ffe837;
+      }
+      .toast {
+        position: fixed;
+        bottom: 20px;
+        left: 50%;
+        transform: translateX(-50%);
+        background: #02576c;
+        color: white;
+        padding: 10px 20px;
+        border-radius: 4px;
+        font-size: 16px;
+        opacity: 0;
+        transition: opacity 0.3s;
+        pointer-events: none;
+        z-index: 1000;
+      }
+      .toast.show {
+        opacity: 1;
+      }
+      .instructions {
+        font-size: 16px;
+        text-align: center;
+        margin-bottom: 20px;
+        color: #ffe837;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="instructions">
+      Paste images (Ctrl+V), drop image files, or click + to paste from clipboard.
```

Comment:
This is intended to be used for mobile, so it'd be nice if there was a way to paste on mobile too, like just having a text input somewhere